app/forms/webauthn_setup_form.rb

```diff
@@ -92,6 +101,21 @@ def safe_response(original_origin)
     false
   end
 
+  def process_authenticator_data_value(data_value)
+    return unless data_value.present?
+    # Ensure that its an int
+    return unless data_value.scan(/\D/).empty?
+    data_int_value = data_value.to_i
+    {
+      up: ActiveModel::Type::Boolean.new.cast(data_int_value & (1 << 0)),
+      uv: ActiveModel::Type::Boolean.new.cast(data_int_value & (1 << 2)),
+      be: ActiveModel::Type::Boolean.new.cast(data_int_value & (1 << 3)),
+      bs: ActiveModel::Type::Boolean.new.cast(data_int_value & (1 << 4)),
+      at: ActiveModel::Type::Boolean.new.cast(data_int_value & (1 << 6)),
```

Comment:
we skip from 4 to 6 here, is there supposed to be a 5?

Reply:
5 is "reserved for future use"

https://w3c.github.io/webauthn/#sctn-authenticator-data

Would probably help to include a link to this somewhere in the code as reference @mdiarra3

Reply:
nvm I just got linked to the docs, there is no 5 😭 https://w3c.github.io/webauthn/#ref-for-authdata-flags-ed